types.d.ts

```diff
@@ -146,10 +146,14 @@ export interface ReleaseStatement {
  *
  * It is an array of `simple_support_statement` objects, but if there
  * is only one of them, the array must be omitted.
+ *
+ * Internally, "mirror" may also be used to represent that the data
+ * should be mirrored from its upstream counterpart.
  */
 export type SupportStatement =
   | SimpleSupportStatement
-  | SimpleSupportStatement[];
+  | SimpleSupportStatement[]
+  | 'mirror';
```

Comment:
Is "mirror" in the exported/released data though?

Reply:
It's not, but I'm adding it in here to prepare for our own internal TypeScript migration!